vpd-tool/meson.options

```diff
@@ -0,0 +1 @@
+option('INVENTORY_JSON_SYM_LINK', type: 'string', value: '/var/lib/vpd/vpd_inventory.json',  description: 'Symbolic link to vpd inventory json.')
```

Comment:
Not needed, it can be taken from parent folder

Reply:
Done